Touches up the admin outfits. 🤳

FlattestGuitar opened the pull request:
No changelog, because this is all backstage. 

Adds SolGov military uniforms, as well as the Representative's uniform - for use in events or away missions.

Or when we swtich to NSS Cyberiad being a SolGov station.

Many thanks to @TravellingMerchant and Dumbdumn5(who I can't tag for some reason)
![](http://i.imgur.com/gEBmlXt.png) ![](http://i.imgur.com/pl33N0k.png) ![](http://i.imgur.com/vDAhmDI.png)

I've rotated around the stuff that's already in the `Select equipment` menu and tried to make it a tad cleaner, as well as added a few loadouts that use the preexisting equipment.

🔧 outfits tweaked:
- Syndicate Nuclear Operative - changed the name to better describe what it is
- Syndicate Officer - stripped their softsuit; gave them a deagle. Is now useful for briefing SITs or other things; not as viable as a combat role (which it was never used as), also gave them Syndicate Officer access, because, y'know, they're a Syndicate Officer
- Space Pirates were replaced with non-spaceproof versions, with a rank of First Mate added in the middle
- "Vox" was renamed to "Vox Raider", as that's what the loadout is


⭐️ outfits added:
- Syndicate Infiltrator - same stuff as SIT members get
- NT Diplomat - swanky diplomatic uniform, white gloves, no weapons
- SolGov Rep
- SolGov Marine - with a GUN
- SolGov Marine Lieutenant - with a nicer uniform and less gun (😢)
- Janitorial ERT member - wears a sexy janitorial ERT rigsuit and is packed full of cleaning supplies, no weapons

Crazylemon64 commented:
I'd rather you not axe "extraneous" outfit datums (they're useful for the outfit application machines) - though, actually, now that I think of it, I'd be OK with it if you added outfit datum {,de}serialization, so that you can set a var to a string to create any uniform you want

tigercat2000 commented:
`Out are the Assassin, Survivor, Dark Lord, Masked Killer, Spy and all the wizards.`

Why? What purpose does it serve to get rid of these? Just because you don't know/can't find a use for them doesn't mean anyone else can't.

Survivor:
 - It's literally designed to be a "bloody torn-up survivor [of a conflict]". An event involving a crashed shuttle could make use of these on the corpses.

Dark Lord:
 - It's a Sith Lord! Lightsaber, robes, equip two players with this and change their esword colors. Lightsaber fight!

Masked Killer:
 - I'll admit this one is really more of a reference to really old admins and their butcher, but it's still useful, it's a dude with a fireaxe, welding mask, and bloody clothes. A "masked killer"!

All of the wizards:
 - Easiest way to equip someone you turn into a wizard. It's annoying as hell to spawn all of the gear by hand for them to use without having the noclothes spell.

KingPhilipIII commented:
I see Dark Lord used frequently actually. Other three not so much.

Kyep commented:
I use the Assassin, Survivor, Masked Killer and Spy regularly. I've seen other people use the Dark Lord, and Wizards, several times.

Norgad commented:
M-Muh voxxy loadout ;v;

fludd12 commented:
I personally use wizard often just because I'm too lazy to spawn the book path

tigercat2000 commented:
> I've also tossed out the Vox loadout, since it didn't make much sense. I'll readd proper raider stuff once the PR for raiders gets merged.

I didn't even see this. This is pointless to remove as well- remove it in your traders PR if you must, but the outfit is for equipping both Vox Primalis and Vox Armalis. It makes perfect sense.

SigholtStarsong commented:
So I guess you unilaterally decided to press on with the same generic sprites?

Necaladun commented:
Assassin is great generic guy in suit with briefcase actually.

All for the new stuff, but all these are somewhat useful. The menu isn't too clogged currently.

Spacemanspark commented:
:+1: to the solgov uniform.
:-1: to the removal of the outfits.

Draconic-saint commented:
If you're that fed up with clogged menu's use folders to sort them.
Don't axe the loadouts please, they're easy and fun to use.

Birdtalon commented:
👍 to adding the uniforms.
👎 to removing the others.

FlattestGuitar commented:
@SigholtStarsong, everyone but you finds the sprites neat, from what I've heard. I also asked on the lore channel, but have received no reply. Feel free to suggest an improvement.

Everyone else, STOP RUINING MY FUN. >:c

I've readded all the outfits I had removed, but moved them slightly lower in the list, so that the Syndicate, NT and more commonly used outfits are near the top.

tigercat2000 commented on code/datums/outfits/outfit_admin.dm:
```diff
@@ -361,572 +460,554 @@
 	if(istype(I))
 		apply_to_card(I, H, list(access_maint_tunnels), "Greytide Leader")
 
-/datum/outfit/admin/greytide/xeno
```
You didn't readd this anywhere.

tigercat2000 commented on code/datums/outfits/outfit_admin.dm:
```diff
@@ -361,572 +460,554 @@
-	H.sec_hud_set_implants()
+		apply_to_card(I, H, get_all_accesses(), "Singuloth Knight")
 
-/datum/outfit/admin/nt_navy_captain
```
Didn't readd this anywhere either.

tigercat2000 commented on code/datums/outfits/outfit_admin.dm:
```diff
@@ -361,572 +460,554 @@
 
-/datum/outfit/admin/assassin
-	name = "Assassin"
+/datum/outfit/admin/janitorial_ert
```
This addition isn't documented in the OP.

tigercat2000 commented on code/datums/outfits/outfit_admin.dm:
```diff
@@ -361,572 +460,554 @@
-
+		apply_to_card(I, H, get_all_accesses(), "Reaper", "syndie")
 
-/datum/outfit/admin/nt_special_ops_formal
```
Also not readded.

tigercat2000 commented on code/datums/outfits/outfit_admin.dm:
```diff
@@ -361,572 +460,554 @@
+/datum/outfit/admin/tournament_gangster //gangster are supposed to fight each other. --rastaf0
+	name = "Tournament Gangster"
 
-/datum/outfit/admin/syndicate/bomber
```
You also didn't readd this.

tigercat2000 commented on code/datums/outfits/outfit_admin.dm:
```diff
@@ -18,193 +18,325 @@
+	if(istype(I))
+		apply_to_card(I, H, list(access_maint_tunnels), name)
+
+/datum/outfit/admin/pirate/first_mate
```
Neither of these pirate outfits are documented in the OP.

tigercat2000 commented on code/datums/outfits/outfit_admin.dm:
```diff
@@ -18,193 +18,325 @@
-		apply_to_card(I, H, list(access_maint_tunnels), "Pirate")
+		apply_to_card(I, H, get_centcom_access("Nanotrasen Navy Officer"), "Nanotrasen Navy Officer")
 
-/datum/outfit/admin/pirate/space
```
Didn't readd this.

tigercat2000 commented on code/datums/outfits/outfit_admin.dm:
```diff
@@ -18,193 +18,325 @@
-	back = /obj/item/weapon/tank/jetpack/oxygen
-	mask = /obj/item/clothing/mask/breath
-	id = /obj/item/weapon/card/id
+/datum/outfit/admin/syndicate/infiltrator
```
Not documented, again.

FlattestGuitar commented:
Whoops. Fixed.

Also, I won't be pasting all of that into the OP. I've changed it to reflect the fact that there are a lof of changes, but there are so many it would be easier to just look through the new code.

Draconic-saint commented:
@FlattestGuitar fuck your fun >:(

KasparoVy commented:
The sprites look utterly fantastic. Should be right now that there's nothing being removed, just this stuff being added + the formatting differences.

TravellingMerchant commented:
@KasparoVy Thanks!

Fox-McCloud commented:
@FlattestGuitar 

Need to list all the new outfits you added.

FlattestGuitar commented:
@Fox-McCloud OP has been updated.

Kyep commented on code/datums/outfits/outfit_admin.dm:
```diff
@@ -18,222 +18,436 @@
-	r_pocket = /obj/item/weapon/kitchen/knife
-	l_hand = /obj/item/weapon/kitchen/knife
-	r_hand = /obj/item/weapon/kitchen/rollingpin
+/datum/outfit/admin/syndicate/officer
```
Not a fan of the changes made to this. https://github.com/ParadiseSS13/Paradise/blob/294a90e9167379af24a47f2f8f01d59ef4c1fb4f/code/modules/admin/verbs/infiltratorteam_syndicate.dm#L117 has the syndi officer loadout, and every item in it is chosen for a reason. The thermal glasses let them see the SIT team through the shutters, opening them at the right time for dramatic effect. The hardsuit is for intimidation and presence - not combat. The 500 TC implant is so that they can spawn a team with 0 TC, and buy items for them to give them specific equipment. The dust implant is a safety measure intended to prevent their gear falling into the wrong hands, should things (e.g: people hijacking the SIT shuttle) happen.

FlattestGuitar replied:
I use the Syndicate Officer loadout almost daily, because the default SIT leader is ugly and doesn't work with a custom character. I've never found the sapce suit to be useful. It only makes sense if you're trying to hide your appearance or go out there and help the SIT, which is silly.

The thermal glasses aren't all that useful (at least in my opinion) and they do not look operator (that's objective).

I've added the 500 TC implant, though, since that's useful.

Kyep commented on code/datums/outfits/outfit_admin.dm:
```diff
@@ -18,222 +18,436 @@
+	shoes = /obj/item/clothing/shoes/syndigaloshes
+	pda = /obj/item/device/pda
+
+/datum/outfit/admin/syndicate/infiltrator/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```
This does not include either a dust implant, or an uplink implant, both of which are critical items for SIT team members. It is also missing a flashlight, and possibly other items.

FlattestGuitar commented:
The infiltrator loadout now matches the actual infiltrators, though it doesn't add them to the HUD. That'd be inconsistent with the behavior of all of the other outfits - wizards, nukies, everything else. I'd rather not make stuff behave weirdly, and if we do spawn SITs like that, we can always add them to the HUD using the traitor panel. ez pz.
